### DISCUSSION TITLE
How to passing props from client to server without schema validation?

### DISCUSSION THREAD

**Discussion by farhantallei** [next-safe-action/next-safe-action#143] [Q&A] [answered]
I'm trying to pass props to a server without schema validation. Any straightforward solutions or advice would be greatly appreciated.

```ts
  // server action
  const schema = zfd.formData({ ... })

  export const updateSomething = authAction(schema, async () => { ... }) // do something
```

```ts
  // client side
  const action = useAction(updateSomething(needToPassingPropsHere))
```

Usually, I use `updateSomething.bind(null, props)`

**Comment by TheEdoRan** [comment, marked as answer]
v7 has [support for bind arguments](https://next.next-safe-action.dev/docs/safe-action-client/bind-arguments).

**Reply by farhantallei** [reply]
Oh, thanks for letting me know about version 7! Didn't know that version 7 was available. Currently, I'm still on version 6.

I'll give it a try later.

**Reply by TheEdoRan** [reply]
You're welcome. I'm waiting for Next.js 14.3.0 stable release to publish v7, because there's no support for React `useActionState` hook in current Next.js stable version.
